src/Microsoft.FeatureManagement/ConfigurationFeatureFlagDefinitionProvider.cs

```diff
@@ -149,6 +152,20 @@ We support
             }
             else
             {
+                string rawRequirementType = configurationSection[RequirementTypeKeyword];
+
+                if (!string.IsNullOrEmpty(rawRequirementType))
+                {
+                    if (!Enum.TryParse(rawRequirementType, true, out RequirementType r))
```

Comment:
We dont need a new variable for RequirementType.
We can do this:
`Enum.TryParse(rawRequirementType, true, out requirementType)`

Then we dont even need to do `requirementType = r` later.